packages/core/test/create-app.js

```diff
@@ -616,4 +617,66 @@ describe('create-app', () => {
       );
     });
   });
+
+  describe('response.content parsing', () => {
```

Comment:
This adds test to ensure that form-mode/shorthand request still work with both JSON and form-urlencoded. We didn't have any tests for it AFAIK :(